### DISCUSSION TITLE
Can I use custom OAuth provider in Clerk?

### DISCUSSION THREAD

**Discussion by sean-ahn** [clerk/javascript#3347] [Support request - DO NOT OPEN HERE] [answered]
Hi there,

I am just curious about if I can use custom providers, such as Kakao ([supported by supabase](https://supabase.com/docs/guides/auth/social-login/auth-kakao)), like auth.js (a.k.a. next-auth).

It would be great if the Clerk supports this feature since Supabase Auth doesn't.

I really appreciate any help you can provide.

**Comment by jescalan** [comment, marked as answer]
We are [working on this actively](https://feedback.clerk.com/roadmap/f9045ac8-0c8e-4f30-b84f-8d551b0767b9) and should have it released within the next few days! Drop a vote on the item on the link here and we'll make sure you get a note when it's released 😁

**Reply by paustint** [reply]
THANK YOU, THANK YOU, THANK YOU FOR FINALLY DELIVERING THIS!
I am now starting my build and can finally move off of Auth0... I have been waiting for this since launch.

**Reply by jescalan** [reply]
So glad to hear this @paustint! Sorry it took us so long. Best of luck with your migration and thrilled to have you as a customer!
